bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -51,6 +50,14 @@ class NavBar extends Component {
   }
 
   handleToggleUserList() {
+    let btn = document.getElementById('UserListBtn')
```

Comment:
You NEVER should touch the DOM by yourself when using React. But if needed use the [RefAPI](https://facebook.github.io/react/docs/refs-and-the-dom.html).

You should just have the `isExpanded` as a prop and pass it from the `NavBarContainer`. Then you can use it on the aria attribute. You dont need to use any DOM method to solve this case.